.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
[Support for macOS users, huh?](https://www.youtube.com/watch?v=CNYemYxN9CA)

Reply:
guilty XD